packages/preview/src/browser/preview-handler.ts

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
it would be nice to reexport APIs via `browser/index.ts`

Reply:
it's already reexported.